src/i18n/locales/de/translation.json

```diff
@@ -270,7 +270,8 @@
           "failed_to_request_sms": "SMS-Code konnte nicht angefordert werden",
           "failed_to_login_sms": "Anmeldung mit SMS-Code fehlgeschlagen",
           "setup_failed": "Einrichtung fehlgeschlagen",
-          "missing_jwt": "JWT fehlt, Anmeldung nicht möglich."
+          "missing_jwt": "JWT fehlt, Anmeldung nicht möglich.",
+          "invalid_jwt": "ungültige JWT übermittelt"
```

Comment:
```suggestion
          "invalid_jwt": "ungültigen JWT übermittelt"
```

imo it's DER token.

Reply:
text is managed in weblate. weblate then opens PRs, but we should edit it in weblate.

Reply:
Currently @jungusemlevve is working on the German translations. Might be good to tag them here to raise awareness.

"Test, test" @jungusemlevve do you get notifications for these comments?

Reply:
@escapedcat Hallo, gibt es Unstimmigkeiten?

Reply:
Schau mal ueber @jankoegel 's Kommentare hier bitte. Da Deutsch eh noch nichts live ist, koennen wir dies trotzdem mergen und Du kannst das ja spaeter korriegieren. Oder?